Source/Readers/UCIFastReader/UCIParser.h

```diff
@@ -9,7 +9,8 @@
 #include <string>
 #include <vector>
 #include <assert.h>
-#include <stdint.h>
+#define __STDC_FORMAT_MACROS
+#include <inttypes.h>
```

Comment:
Don't think, this replacement is really called for here (and in a few other places as well)